kube/aks/README.md

```diff
@@ -29,11 +29,20 @@ portal](https://portal.azure.com/#create/Microsoft.AKS).  The standard settings
 are fine for this use-case although you might need to adjust a few things to
 match the scale of your particular deployment.
 
-## Atlas MongoDB account
 
-This AKS reference deployment relies on an Atlas account for MongoDB in order
-to keep everything in the Cloud and application setup to the bare minimal.  As
-such, please [create an Atlas
+## MongoDB/Atlas database
+
+### Self-hosted MongoDB
+
+By default script api-initial-deploy.sh install self-hosted MongoDB,
+and store data in PVC.
+Adjust size of PVC in deploy/pvc.yaml.
```

Comment:
I thought we were planning to rely on Atlas for the production deployment?  Having "self-hosted" MongoDB only applies to the vendor-agnostic Kubernetes files with e.g. MiniKube.

Reply:
Production yes, but for staging we can use self-hosted MongoDB (also to reduce costs), so this script planned to be used for both options.

Reply:
For staging we have docker-compose.  This seems to be mixing a lot of things up I think.